pants.toml

```diff
@@ -189,6 +189,7 @@ extra_env_vars = [
   "PATH",
   # We'd always like complete backtraces in tests.
   "RUST_BACKTRACE=1",
+  "_PANTS_VERSION_OVERRIDE=3.0.0",
```

Comment:
Why `3.0.0`..? guessing it's arbitrary, to me would feel less strange to pick a 2.x version as that's where we are currently.

Reply:
Yeah it's an arbitrary number which also acts like a sentinel.

I'll add a comment.